code/include/draw.h

```diff
@@ -53,6 +53,7 @@
 #define COLOR_GREEN         RGB8(0x00, 0x9E, 0x73)
 #define COLOR_BLUE          RGB8(0x56, 0xB4, 0xE9)
 #define COLOR_ORANGE        RGB8(0xE6, 0x9F, 0x00)
+#define COLOR_YELLOW        RGB8(0xFF, 0xFF, 0x00)

```

Comment:
This should be adjusted for colorblindness, see acceptable colors here: https://github.com/gamestabled/OoT3D_Randomizer/pull/435#issuecomment-992089545
The closest, which is still very yellow, would be:
```suggestion
#define COLOR_YELLOW        RGB8(0xF0, 0xE4, 0x42)
```